python/paddle/fluid/layers/nn.py

```diff
@@ -1486,7 +1486,9 @@ def batch_norm(input,
                in_place=False,
                name=None,
                moving_mean_name=None,
-               moving_variance_name=None):
+               moving_variance_name=None,
+               average_mean=True,
+               average_variance=True):
```

Comment:
1. default by False
2.  average_mean, average_variance -> do_model_average_for_mean_and_var?
 
They can share the same flag.

Reply:
Done.